app/controllers/admin/homes_controller.rb

```diff
@@ -7,4 +7,20 @@ def index
       render file: "#{Rails.root}/public/404.html", status: 404
     end
   end
+
+  def suspend
+    home_to_suspend = Home.find(params[:id])
+    home_to_suspend.suspend if current_admin?
+    @homes = Home.all
```

Comment:
Unnecessary since this action just redirects, @homes isnt need as there is no page for it to be used on